test/e2e/nodes_test.go

```diff
@@ -27,9 +29,22 @@ var _ = Describe("Nodes", func() {
 			for _, node := range nodes {
 				key := types.NamespacedName{Namespace: namespace, Name: node}
 				nodeNetworkState := nodeNetworkState(key)
-				//TODO: exec nmstatectl is not in place let's just compare
-				//      with the stuff we have harcoded there
-				Expect(nodeNetworkState.Status.CurrentState).To(MatchYAML("interfaces: []"))
+				currentStateYaml := nodeNetworkState.Status.CurrentState
+				Expect(currentStateYaml).ToNot(BeEmpty())
+				var currentState map[string][]map[string]interface{}
+				err := yaml.Unmarshal(currentStateYaml, &currentState)
+				Expect(err).ToNot(HaveOccurred())
+				interfaces := currentState["interfaces"]
+				Expect(interfaces).ToNot(BeEmpty())
+				obtainedInterfaces := interfacesName(interfaces)
+				Expect(obtainedInterfaces).To(SatisfyAll(
+					ContainElement("lo"),
+					ContainElement("cni0"),
+					ContainElement("eth0"),
+					ContainElement("eth1"),
+					ContainElement("flannel.1"),
```

Comment:
please try it on both openshift and kubernetes. i suspect you won't find flannel on openshift

Reply:
Maybe we can discover were are we running and test according.

Reply:
I see no benefit in checking for flannel.1. Just check eth0 and eth1 (we are still expecting those tests to be executed against local cluster. Later we can use nmstate to create a set of interfaces in BeforeAll and then verify them.

Reply:
Done.